restservice/src/test/resources/test-local.properties.TEMPLATE

```diff
@@ -12,10 +12,6 @@ changelog.file=src\\test\\resources\\liquibase\\db.changelog-create.xml
 # SendGrid
 sendgrid.api.key=SEND_GRID_TOKEN
 
-# EventService
-event.filter.range.in.days=7
```

Comment:
@IgorMaksymov what will be with tests if somebody changes default variables in future?